components/playlist/browser/mime_util.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_PLAYLIST_BROWSER_MIME_UTIL_H_
+#define BRAVE_COMPONENTS_PLAYLIST_BROWSER_MIME_UTIL_H_
+
+#include <optional>
+#include <string>
+
+#include "base/files/file_path.h"
+
+namespace playlist::mime_util {
```

Comment:
might be worth a comment explaining why we can't use the Chromium function

Reply:
Good call!